test/e2e/multihoming.go

```diff
@@ -254,12 +261,129 @@ var _ = Describe("Multi Homing", func() {
 				},
 			),
 		)
+
+		const (
+			clientPodName     = "client-pod"
+			clientIPOffset    = 100
+			serverIPOffset    = 102
+			port              = 9000
+			workerOneNodeName = "ovn-worker"
+			workerTwoNodeName = "ovn-worker2"
+		)
+
+		ginkgo.DescribeTable("attached to a localnet network mapped to breth0",
+
+			func(netConfigParams networkAttachmentConfigParams, clientPodConfig, serverPodConfig podConfiguration) {
+
+				netConfig := newNetworkAttachmentConfig(networkAttachmentConfigParams{
+					name:      secondaryNetworkName,
+					namespace: f.Namespace.Name,
+					topology:  "localnet",
+				})
+				if clientPodConfig.namespace == "" {
+					clientPodConfig.namespace = f.Namespace.Name
+				}
+				if serverPodConfig.namespace == "" {
+					serverPodConfig.namespace = f.Namespace.Name
+				}
+
+				By("setting up the localnet underlay")
+				pods := ovsPods(cs)
+				Expect(pods).NotTo(BeEmpty())
+
+				Expect(setupUnderlayOnDefaultBridge(pods, netConfig)).To(Succeed())
```

Comment:
where do you tear this mapping down ? IIUC, it is being left configured.

Reply:
Sure! Let me fix that also for the case of secondary bridge case... I've just noticed that removing the bridge is not enough, we should explicitly set the desired bridge mapping.

Reply:
We're now tearing down the underlay using a `defer`ed call.